### ISSUE TITLE
Failed to install NuGet package when installing on project with log4net 1.2.13

### ISSUE THREAD

**Issue by jpierson** [plmwong/log4net-raygun#10]
It would be awesome if the nuget package would allow versions greater than 1.2.10 or at least make a package that specifically targets 1.2.13. Is this possible?

Attempting to resolve dependency 'log4net (= 1.2.10)'.
Attempting to resolve dependency 'Mindscape.Raygun4Net (≥ 5.0.0)'.
Installing 'log4net 1.2.10'.
Successfully installed 'log4net 1.2.10'.
Installing 'log4net.1.2.10.Raygun 4.1.0.25085'.
Successfully installed 'log4net.1.2.10.Raygun 4.1.0.25085'.
Install failed. Rolling back...
Already referencing a newer version of 'log4net'.

**Comment by plmwong** [comment]
Hmm, curious, the main https://www.nuget.org/packages/log4net.Raygun/ packageshould allow any version of log4net above 2.0.2 (aka 1.2.12). And the alternate https://www.nuget.org/packages/log4net.1.2.10.Raygun/ package should be the only one locked to 1.2.10.

I'll double check this.

**Comment by plmwong** [comment]
I just tested this out by adding in `log4net [1.2.13]` into a solution first, and then adding in `log4net.Raygun` after, which was successful. Could you double check that you are trying to add the main `log4net.Raygun` package from NuGet?

```
------- Installing...log4net 2.0.3 -------
Installing 'log4net 2.0.3'.
Added file 'log4net.dll' to folder 'log4net.2.0.3\lib\net10-full'.
Added file 'log4net.xml' to folder 'log4net.2.0.3\lib\net10-full'.
Added file 'log4net.dll' to folder 'log4net.2.0.3\lib\net11-full'.
Added file 'log4net.xml' to folder 'log4net.2.0.3\lib\net11-full'.
Added file 'log4net.dll' to folder 'log4net.2.0.3\lib\net20-full'.
Added file 'log4net.xml' to folder 'log4net.2.0.3\lib\net20-full'.
Added file 'log4net.dll' to folder 'log4net.2.0.3\lib\net35-client'.
Added file 'log4net.xml' to folder 'log4net.2.0.3\lib\net35-client'.
Added file 'log4net.dll' to folder 'log4net.2.0.3\lib\net35-full'.
Added file 'log4net.xml' to folder 'log4net.2.0.3\lib\net35-full'.
Added file 'log4net.dll' to folder 'log4net.2.0.3\lib\net40-client'.
Added file 'log4net.xml' to folder 'log4net.2.0.3\lib\net40-client'.
Added file 'log4net.dll' to folder 'log4net.2.0.3\lib\net40-full'.
Added file 'log4net.xml' to folder 'log4net.2.0.3\lib\net40-full'.
Added file 'log4net.2.0.3.nupkg' to folder 'log4net.2.0.3'.
Successfully installed 'log4net 2.0.3'.
Adding 'log4net 2.0.3' to raygun-test.
For adding package 'log4net 2.0.3' to project 'raygun-test' that targets 'net45',
>> Assembly references are being added from 'lib\net40-full'
Added reference 'log4net' to project 'raygun-test'
Added file 'packages.config'.
Added file 'packages.config' to project 'raygun-test'
Successfully added 'log4net 2.0.3' to raygun-test.
==============================

------- Installing...log4net.Raygun 4.1.0.21896 -------
Attempting to resolve dependency 'log4net (≥ 2.0.2)'.
Attempting to resolve dependency 'Mindscape.Raygun4Net (≥ 5.0.0)'.
Installing 'Mindscape.Raygun4Net 5.0.0'.
Added file 'Mindscape.Raygun4Net.Xamarin.Android.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\MonoAndroid2.2'.
Added file 'Mindscape.Raygun4Net.Xamarin.Android.pdb' to folder 'Mindscape.Raygun4Net.5.0.0\lib\MonoAndroid2.2'.
Added file 'Mindscape.Raygun4Net.Xamarin.iOS.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\MonoTouch'.
Added file 'Mindscape.Raygun4Net.Xamarin.iOS.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\MonoTouch4.0'.
Added file 'Mindscape.Raygun4Net.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\net20'.
Added file 'Mindscape.Raygun4Net.pdb' to folder 'Mindscape.Raygun4Net.5.0.0\lib\net20'.
Added file 'Mindscape.Raygun4Net.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\net35-client'.
Added file 'Mindscape.Raygun4Net.pdb' to folder 'Mindscape.Raygun4Net.5.0.0\lib\net35-client'.
Added file 'Mindscape.Raygun4Net.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\net35'.
Added file 'Mindscape.Raygun4Net.pdb' to folder 'Mindscape.Raygun4Net.5.0.0\lib\net35'.
Added file 'Mindscape.Raygun4Net.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\net40-client'.
Added file 'Mindscape.Raygun4Net.pdb' to folder 'Mindscape.Raygun4Net.5.0.0\lib\net40-client'.
Added file 'Mindscape.Raygun4Net.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\net40'.
Added file 'Mindscape.Raygun4Net.pdb' to folder 'Mindscape.Raygun4Net.5.0.0\lib\net40'.
Added file 'Mindscape.Raygun4Net4.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\net40'.
Added file 'Mindscape.Raygun4Net4.pdb' to folder 'Mindscape.Raygun4Net.5.0.0\lib\net40'.
Added file 'Mindscape.Raygun4Net.WindowsStore.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\portable-net45+win81+wpa81+windows81'.
Added file 'Mindscape.Raygun4Net.WindowsStore.pdb' to folder 'Mindscape.Raygun4Net.5.0.0\lib\portable-net45+win81+wpa81+windows81'.
Added file 'Mindscape.Raygun4Net.WinRT.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\portable-windows8'.
Added file 'Mindscape.Raygun4Net.WinRT.pdb' to folder 'Mindscape.Raygun4Net.5.0.0\lib\portable-windows8'.
Added file 'Mindscape.Raygun4Net.WindowsPhone.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\windowsphone8'.
Added file 'Mindscape.Raygun4Net.WindowsPhone.pdb' to folder 'Mindscape.Raygun4Net.5.0.0\lib\windowsphone8'.
Added file 'Mindscape.Raygun4Net.WindowsPhone.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\wp71'.
Added file 'Mindscape.Raygun4Net.WindowsPhone.pdb' to folder 'Mindscape.Raygun4Net.5.0.0\lib\wp71'.
Added file 'Mindscape.Raygun4Net.Xamarin.iOS.Unified.dll' to folder 'Mindscape.Raygun4Net.5.0.0\lib\Xamarin.iOS10'.
Added file 'readme.txt' to folder 'Mindscape.Raygun4Net.5.0.0'.
Added file 'Mindscape.Raygun4Net.5.0.0.nupkg' to folder 'Mindscape.Raygun4Net.5.0.0'.
Successfully installed 'Mindscape.Raygun4Net 5.0.0'.
Installing 'log4net.Raygun 4.1.0.21896'.
Added file 'log4net.Raygun.Core.dll' to folder 'log4net.Raygun.4.1.0.21896\lib\net40'.
Added file 'log4net.Raygun.dll' to folder 'log4net.Raygun.4.1.0.21896\lib\net40'.
Added file 'log4net.Raygun.Core.dll' to folder 'log4net.Raygun.4.1.0.21896\lib\net45'.
Added file 'log4net.Raygun.dll' to folder 'log4net.Raygun.4.1.0.21896\lib\net45'.
Added file 'log4net.Raygun.4.1.0.21896.nupkg' to folder 'log4net.Raygun.4.1.0.21896'.
Successfully installed 'log4net.Raygun 4.1.0.21896'.
Adding 'Mindscape.Raygun4Net 5.0.0' to raygun-test.
For adding package 'Mindscape.Raygun4Net 5.0.0' to project 'raygun-test' that targets 'net45',
>> Assembly references are being added from 'lib\net40'
Added reference 'Mindscape.Raygun4Net' to project 'raygun-test'
Added reference 'Mindscape.Raygun4Net4' to project 'raygun-test'
Added file 'packages.config'.
Successfully added 'Mindscape.Raygun4Net 5.0.0' to raygun-test.
Adding 'log4net.Raygun 4.1.0.21896' to raygun-test.
For adding package 'log4net.Raygun 4.1.0.21896' to project 'raygun-test' that targets 'net45',
>> Assembly references are being added from 'lib\net45'
Added reference 'log4net.Raygun.Core' to project 'raygun-test'
Added reference 'log4net.Raygun' to project 'raygun-test'
Added file 'packages.config'.
Successfully added 'log4net.Raygun 4.1.0.21896' to raygun-test.
==============================
```

**Comment by jpierson** [comment]
Thank you for pointing it out to me but I was unaware that 1.2.13 was included in the 2.0.2 >= version. I'm familiar with Log4Net but haven't used it much and was adding raygun support into an existing project. With a quick look at the version numbers I assumed that 1.2.13 < 2.0.2 and thus I would not be able to use the latest package, I was wrong. Thanks for pointing me in the right direction.

**Comment by plmwong** [comment]
@jpierson Not a problem, glad I could help :+1:  

Yeah, after 1.2.10, along with changing their public key, log4net bumped up their versioning to start at 2.0.0. But they're still more commonly referred to by the continuation of the old version numbers.

Which means the package named `log4net [1.2.11]` is actually version `2.0.0`; `log4net [1.2.12]` is actually version `2.0.1` _or_ `2.0.2`, `log4net [1.2.13]` is `2.0.3` etc.

Terribly confusing.
